testbed/testbed/exporters.go

```diff
@@ -105,3 +133,54 @@ func (je *jaegerExporter) GetCollectorPort() int {
 func (je *jaegerExporter) ProtocolName() string {
 	return "jaeger"
 }
+
+// ocMetricsExporter implements MetricExporter for OpenCensus metrics protocol.
+type ocMetricsExporter struct {
+	exporter exporter.MetricsExporter
+	port     int
+}
+
+// Create a new OpenCensus metric protocol exporter.
+func NewOcMetricExporter(port int) *ocMetricsExporter {
```

Comment:
doc should start with NewOcMetricExporter.

Reply:
fixed.